script/new-artifact.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
Best practice (AFAIK) is to use the `/usr/bin/env` shebang so that it honors any `PATH` configurations to the desired executable (`sh`, in this case) rather than explicitly relying on it existing at the path `/bin/sh`.

```suggestion
#!/usr/bin/env sh
```